frontend/src/components/pages/CreateReviewPage.tsx

```diff
@@ -0,0 +1,22 @@
+import React from "react";
+
+import AddNewInput from "../AddNewInput";
+import Tags from "../Tags";
+
+const CreateReview = (): React.ReactElement => {
```

Comment:
Is there a way to do it so that the CreateReview has the state: i.e. array of tags and array of input fields? It's because when we're actually doing the page, we want the actual state to all be stored in this create review page so we can submit all that data at once with a submit button